server/src/internalClusterTest/java/org/elasticsearch/discovery/StableMasterDisruptionIT.java

```diff
@@ -610,7 +597,14 @@ public void testNoQuorum() throws Exception {
         networkDisconnect.startDisrupting();
         internalCluster().stopNode(firstMasterNode);
         for (String nonActiveMasterNode : nonActiveMasterNodes) {
-            assertMasterStability(internalCluster().client(nonActiveMasterNode), HealthStatus.RED, "unable to form a quorum");
+            assertMasterStability(
+                internalCluster().client(nonActiveMasterNode),
+                HealthStatus.RED,
+                anyOf(
+                    containsString("unable to form a quorum"),
+                    containsString("No master node observed in the last 1s") // happens if master node has not replied within 1s
```

Comment:
I think the important part of the error string here (and the part indicating that it's a timing thing) is `and the cause has not been determined.` Mind changing the string to that? The `No master node observed in the last 1s` prefix would potentially be returned by several different problems (and we don't expect to see any of them here).